### ISSUE TITLE
false positive: ex-info with nil data raises error (ClojureScript)

### ISSUE THREAD

**Issue by svenlaater** [clj-kondo/clj-kondo#2683]
We have a few ClojureScript test cases with `are` macro which end up with `ex-info` function calls where `data` is `nil`. It works perfectly fine but raises a clj-kondo complaint.

It can be easily tested w/ adding `(ex-info "test" nil)` to anywhere in ClojureScript codebase but here's an example.

```clojure
erp.integrations.sentry.impl-test>(ex-info "ok" nil)
#error {:message "ok"}
erp.integrations.sentry.impl-test>
```

```sh
clj -M:clj-kondo .
./test/cljs/erp/integrations/sentry/impl_test.cljs:113:26: error: Expected: map, received: nil.
linting took 5542ms, errors: 1, warnings: 0
```

This does not seem to a problem in Clojure. Only ClojureScript.

**version**

2025.10.23

**Comment by borkdude** [comment]
This is because on Clojure it raises an error and the type definition here is the same for CLJ and CLJS. I guess it's an interesting differences between CLJ and CLJS that may not be intentional.

**Comment by borkdude** [comment]
Oh wait, it got "fixed" in Clojure 1.12! :) https://clojure.atlassian.net/browse/CLJ-2640
